components/has/base/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ resources:
 - allow-argocd-to-manage.yaml
 - argocd-permissions.yaml
 - https://github.com/redhat-appstudio/application-service/config/default?ref=b766bf7b930a5cdc63d249ef69cfd653f555877c
+- https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=b766bf7b930a5cdc63d249ef69cfd653f555877c
```

Comment:
no worries, I need to pick up the latest changes in the HAS repo  before I can get this in.  

> Does the commit SHA need to be auto-updated upon merges to HAS, like we do for the above line?

 Yes, I will need to make changes to enable auto-updating the HAS side once this PR gets in.  
 
>  Likewise, why not link it in the default overlays?

I wasn't sure if it should be enabled under the default kustomization file.  I have the [prometheus folder commented out](https://github.com/redhat-appstudio/application-service/blob/main/config/default/kustomization.yaml#L24) because I didn't want it to affect local deployments since local CRC clusters do not support the monitoring stack